bigbluebutton-html5/imports/ui/components/app/styles.scss

```diff
@@ -137,25 +137,30 @@ $bars-padding: $lg-padding-x - .45rem; // -.45 so user-list and chat title is al
   }
 }
 
-.sidebar {
-  @extend %full-page;
-  z-index: 4;
-
-  @include mq($medium-up) {
-    flex: 0 15vw;
-    order: 2;
-  }
-}
-
 .media {
   @extend %full-page;
   flex: 1 100%;
   order: 2;
 
+  flex-direction: row;
+  position: relative;
+
   @include mq($small-only) {
     padding-bottom: $actionsbar-height;
     margin-bottom: $actionsbar-height;
-    position: relative;
+  }
+
+  @include mq($portrait) {
+    flex-direction: column;
```

Comment:
It switches layout here, but it's always small:

![layout](https://cloud.githubusercontent.com/assets/9299997/26016714/4074a1f0-3734-11e7-9494-af14dceb825d.JPG)